change/@microsoft-fast-foundation-61f51210-4c16-4b3f-8984-f6a68f6e54d8.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
@EisenbergEffect I don't think this is exported from the package yet, so this would technically be a "none" change as it doesn't affect the distributed packages.

With that said, we do have a *request* to export this from our packages for use which would support the `minor` version bump denoted here. Do we want to do that or should we change this (and then the `dependentChangeType`) to none?

Reply:
I don't think we should do that. We shouldn't really have a testing-specific function as part of a runtime library. We need to extract that to its own thing, which has implications (see Next Steps).

I can change it to "none" though.